### PR TITLE
Fix: FIT-580. removed acceptance tests from the rollback block.

### PR DESCRIPTION
**Pull request by LotharII**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/FIT-580

#### Brief feature description
- removed acceptance tests from the rollback block.

### CODE REVIEW

File: JenkinsDeploy [hunk 1]
```diff
@@ -33,10 +33,13 @@ def deployWithSmoke(environment){
       deployToStage(environment, env.version)
       updateManifestStage(environment, env.version)
       if (environment == 'integration')  {
-          smokeTest(environment, env.version)
-          integrationAcceptenceTest(environment, env.version)
+        smokeTest(environment, env.version)
       }
     }
+    if (environment == 'integration')  {
```
Comment (kelfink) [inline JenkinsDeploy]: Ah.  This is a good addition.  wrapping it in the test for integration is probably necessary.  Unfortunately I just merged my simpler solution. If you have time to resolve the (my) conflict I'd like to add this change as well.

Comment (LotharII) [reply JenkinsDeploy]: Sure, thanks.

Comment (LotharII) [reply JenkinsDeploy]: I will just create a new PR with wrapping acceptance tests call.

Comment (kelfink) [reply JenkinsDeploy]: Well, up to you.  I resolved the conflict.  If that looks good, go ahead and merge.

Comment (kelfink) [reply JenkinsDeploy]: Oh my merge isn't good enough.  I put the full integration test run outside the node.  It's no good.  I'll try another PR.

**Pull request merged**